### ISSUE TITLE
You don't need the softmax for inference

### ISSUE THREAD

**Issue by harpone** [Makalfo/SnarkyNet-MNIST-Digits#2]
You can just take the argmax of the logit (the linear input to the softmax). The argmax will be the same as if you took the argmax of the softmax, hence same class prediction.
